server/src/main/java/org/opensearch/action/search/SearchRequestContext.java

```diff
@@ -141,6 +142,18 @@ public List<TaskResourceInfo> getPhaseResourceUsage() {
     public SearchRequest getRequest() {
         return searchRequest;
     }
+
+    public void successfulSearchShardIndices(List<String> successfulSearchShardIndices) {
+        this.successfulSearchShardIndices = successfulSearchShardIndices;
+    }
```

Comment:
Should be `setSuccessfulSearchShardIndices`, since this method is not part of Builder?

Reply:
Changed